api/src/test/java/uk/gov/dhsc/htbhf/claimant/PaymentCycleIntegrationTests.java

```diff
@@ -67,6 +67,8 @@
     private static final int CARD_BALANCE_IN_PENCE_BEFORE_DEPOSIT = 88;
     private static final LocalDate NOT_PREGNANT = null;
     private static final String CARD_ACCOUNT_ID = UUID.randomUUID().toString();
+    
+    // TODO HTBHF-2475 - update integration tests to confirm currentIdentityAndEligibilityResponse is updated, once we are calling v3 eligibility service
```

Comment:
Shouldn't this be a part of this PR seeing as the functionality is being added, otherwise it isn't integration tested?

Reply:
We can't add that check until we've updated the scheduled services to use v3, at least not easily